sandbox/shantanu/postgres-db.tf

```
@@ -0,0 +1,21 @@
+
+resource "azurerm_postgresql_server" "test-pg-db" {
+  name                = "test-pg-psqlserver"
+  resource_group_name = azurerm_resource_group.sandbox.name
+  location            = azurerm_resource_group.sandbox.location
+
+  administrator_login          = "myadmin"
+  administrator_login_password = "myadm!n"
+
+  sku_name   = "GP_Gen5_4"
```

You can try with basic size
```suggestion
  sku_name   = "B_Gen4_1"
```